enterprise/admin/__init__.py

```diff
@@ -320,17 +322,57 @@ def get_search_results(self, request, queryset, search_term):
 
         return queryset, use_distinct
 
-    def enrolled_courses(self, enterprise_customer_user):
+    def enterprise_enrollments(self, enterprise_customer_user):
         """
-        Return a string representing the courses a given EnterpriseCustomerUser is enrolled in
+        Return a string representing a given EnterpriseCustomerUser's enterprise course enrollments
 
         Args:
             enterprise_customer_user: The instance of EnterpriseCustomerUser
                 being rendered with this admin form.
         """
-        courses_string = mark_safe(self.get_enrolled_course_string(enterprise_customer_user))
+        enterprise_course_ids = self._get_enterprise_course_enrollments(enterprise_customer_user)
+        courses_string = mark_safe(self.get_enrolled_course_string(enterprise_course_ids))
         return courses_string or 'None'
 
+    def other_enrollments(self, enterprise_customer_user):
+        """
+        Return a string representing a given EnterpriseCustomerUser's non-enterprise course enrollments
+
+        Args:
+            enterprise_customer_user: The instance of EnterpriseCustomerUser
+                being rendered with this admin form.
+        """
+        all_course_ids = self._get_all_enrollments(enterprise_customer_user)
+        enterprise_course_ids = self._get_enterprise_course_enrollments(enterprise_customer_user)
+        # remove overlapping enterprise enrollments from all enrollments
+        course_ids = set(all_course_ids) - set(enterprise_course_ids)
```

Comment:
Not entirely sure if this (i.e., removing the enterprise enrollments from all enrollments) is the best way to find/retrieve non-enterprise course enrollments. Definitely open to feedback here 😃

Reply:
Makes sense to me